sdk/src/client/node_api/core/routes.rs

```diff
@@ -93,33 +93,30 @@ impl ClientInner {
 
     // Blocks routes.
 
-    /// Returns tips that are ideal for attaching a block.
-    /// GET /api/core/v3/tips
-    pub async fn get_tips(&self) -> Result<Vec<BlockId>> {
-        let path = "api/core/v3/tips";
+    /// Returns information that are ideal for attaching a block.
```

Comment:
is ideal; or maybe better "Return information that is used to attach a block in the network"?

Reply:
FYI, this is the documentation from the TIP so feel free to also comment there 🙏🏻

Reply:
Commented on the tip